README.md

```diff
@@ -156,7 +156,7 @@ ensure that
 --- | --- | ---
 1 | passwordless `sudo` works for \<user\> | all machines
 2 | \<user\> can `ssh` from \<origin\> to other machines | \<origin\>
-3 | `cortx-hare` and `cortx-s3server` RPMs are installed | all machines
+3 | `cortx-hare` and `cortx-rgw` RPMs are installed | all machines
```

Comment:
I would suggest to reword it as, `motr client packages, e.g. cortx-rgw, cortx-s3server, etc.`